### DISCUSSION TITLE
Can't change highlight syntax on lazy loading

### DISCUSSION THREAD

**Discussion by AndreJesusBrito** [NvChad/NvChad#1608] [Q&A] [answered]
Hello,

I want to change the syntax highlighting for certain words
so I have in my chadrc:
```lua
  hl_override = {
    -- Git Editor Signs
    DiffAdd = { fg = "git_green" },
    DiffChange = { fg = "git_yellow" },
    DiffModified = { fg = "git_yellow" },
    DiffRemoved = { fg = "git_red" },


    Repeat = { fg = "base0E" },
    Operator = { fg = "base0E" },
    Statement = { fg = "base0E" },
    Function = { fg = "base0E" },
    Constant = { fg = "base0E" },
    Exception = { fg = "base0E" },

    htmlTag = { fg = "white" },
    htmlTagName = { fg = "base08" },
  },
```

It works fine, except for the html.
Maybe its because the html is lazy loaded and so my configuration gets overridden.

Is there a way to, for example, put this options in a separated file so it can also be lazy loaded after the plugin?

Thank you

**Comment by siduck** [comment, marked as answer]
@AndreJesusBrito because htmltag and htmltagname arent even defined by base46. So add them in hl_add table 

```lua
M.ui = {
  hl_add = {
    htmlTag = { fg = "white" },
    htmlTagName = { fg = "blue" },
    }
 }
 ```   
 
 And color names from base30 table are only allowed like white, blue, statusline_bg. Dont use color names from base16 table.

**Reply by AndreJesusBrito** [reply]
Thank you, @siduck, that works!
Should I add more colors to the base30 table? how can I do truecolor?

**Reply by siduck** [reply]
@AndreJesusBrito 

```lua
M.ui = {

   changed_themes = {
      onedark = {
         base_16 = {
            base00 = "#mycol",
         },
         base_30 = {
            red = "#mycol",
            white = "#mycol",
            one_bg = "#mycol",
            newColor = "#mycol" -- so you can add your new color here too!
         },
      },

      nord = {
         -- and so on!
      },
      gruvchad = { -- so on!}
   },
}
```
https://nvchad.com/config/theming#override-specific-colors-in-themes
